runtime/trappist/src/xcm_config.rs

```diff
@@ -97,26 +106,21 @@ pub type LocalAssetTransactor = CurrencyAdapter<
 	// Our chain's account ID type (we can't get away without mentioning it explicitly):
 	AccountId,
 	// We don't track any teleports of `Balances`.
-	(),
+	CheckAccount,
```

Comment:
Is this to support the use case with the AH ?

Reply:
No, however I would do it in a different PR

Reply:
I think it initially came from [here](https://github.com/paritytech/trappist/commit/6cd2ee699dcf7851a8dbb96eccf217f8459e75ab#diff-762c8a6849a94f93f63b9751aba00bbdb14ad9f5794eb7d14c964271bf1fbe68R101)

Reply:
> Is this to support the use case with the AH ?

Correct, this needed to keep track of the in/out coming teleports for Trappist.

 To be honest when we add this one we were not very mindful why we were doing it , but now with the AH use case it's 100% clear and needed.

Reply:
@hbulgarini Let's at least update the comment then, because it is not inline anymore with the change.

Reply:
> @hbulgarini Let's at least update the comment then, because it is not inline anymore with the change.

Actually, i think the configuration was wrong. The only place where the `CheckAccount` was needed is on the CurrencyTransactor that is the one that takes care for the HOP teleport to AH (teleport of native token).
Then for FungibleTransactors (meaning assets) this is not required since for the moment we don't plan to teleport any asset to any location.